valhalla/baldr/errorcode_util.h

```diff
@@ -84,10 +84,6 @@ namespace baldr {
     // loki project 1xx
     {100,"Failed to parse json request"},
     {101,"Try a POST or GET request instead"},
-    {102,"The config actions for Loki are incorrectly loaded"},
-    {103,"Missing max_locations configuration"},
-    {104,"Missing max_distance configuration"},
-    {105,"Path action not supported"},
```

Comment:
these were used in the constructor for the worker. so they'll never become http responses. instead i just threw them as runtime error since they are all erroneous config related